[2023-09-20T08:08:58.446+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.create_chart manual__2023-09-20T08:08:55.633327+00:00 [queued]>
[2023-09-20T08:08:58.449+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.create_chart manual__2023-09-20T08:08:55.633327+00:00 [queued]>
[2023-09-20T08:08:58.449+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-20T08:08:58.455+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonVirtualenvOperator): create_chart> on 2023-09-20 08:08:55.633327+00:00
[2023-09-20T08:08:58.459+0000] {standard_task_runner.py:57} INFO - Started process 4216 to run task
[2023-09-20T08:08:58.462+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'create_chart', 'manual__2023-09-20T08:08:55.633327+00:00', '--job-id', '97', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp48yu5ej2']
[2023-09-20T08:08:58.462+0000] {standard_task_runner.py:85} INFO - Job 97: Subtask create_chart
[2023-09-20T08:08:58.488+0000] {task_command.py:415} INFO - Running <TaskInstance: weather_data.create_chart manual__2023-09-20T08:08:55.633327+00:00 [running]> on host 90bb52186339
[2023-09-20T08:08:58.524+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='create_chart' AIRFLOW_CTX_EXECUTION_DATE='2023-09-20T08:08:55.633327+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-20T08:08:55.633327+00:00'
[2023-09-20T08:08:58.527+0000] {process_utils.py:182} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venv3nli_f_u --system-site-packages
[2023-09-20T08:08:58.534+0000] {process_utils.py:186} INFO - Output:
[2023-09-20T08:08:59.114+0000] {process_utils.py:190} INFO - created virtual environment CPython3.8.18.final.0-64 in 162ms
[2023-09-20T08:08:59.114+0000] {process_utils.py:190} INFO -   creator CPython3Posix(dest=/tmp/venv3nli_f_u, clear=False, no_vcs_ignore=False, global=True)
[2023-09-20T08:08:59.115+0000] {process_utils.py:190} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2023-09-20T08:08:59.115+0000] {process_utils.py:190} INFO -     added seed packages: pip==23.2.1, setuptools==68.1.2, wheel==0.41.2
[2023-09-20T08:08:59.115+0000] {process_utils.py:190} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2023-09-20T08:08:59.135+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venv3nli_f_u/bin/pip install -r /tmp/venv3nli_f_u/requirements.txt
[2023-09-20T08:08:59.142+0000] {process_utils.py:186} INFO - Output:
[2023-09-20T08:08:59.984+0000] {process_utils.py:190} INFO - Requirement already satisfied: pandas in /home/***/.local/lib/python3.8/site-packages (from -r /tmp/venv3nli_f_u/requirements.txt (line 1)) (2.0.3)
[2023-09-20T08:09:00.281+0000] {process_utils.py:190} INFO - Collecting matplotlib (from -r /tmp/venv3nli_f_u/requirements.txt (line 2))
[2023-09-20T08:09:00.282+0000] {process_utils.py:190} INFO -   Obtaining dependency information for matplotlib from https://files.pythonhosted.org/packages/84/76/9267af394cb88ce5245b5a9d328ee90ccd5710a1d08b0ec363c3a86cd992/matplotlib-3.7.3-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata
[2023-09-20T08:09:00.283+0000] {process_utils.py:190} INFO -   Using cached matplotlib-3.7.3-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (5.7 kB)
[2023-09-20T08:09:00.322+0000] {process_utils.py:190} INFO - Requirement already satisfied: python-dateutil>=2.8.2 in /home/***/.local/lib/python3.8/site-packages (from pandas->-r /tmp/venv3nli_f_u/requirements.txt (line 1)) (2.8.2)
[2023-09-20T08:09:00.323+0000] {process_utils.py:190} INFO - Requirement already satisfied: pytz>=2020.1 in /home/***/.local/lib/python3.8/site-packages (from pandas->-r /tmp/venv3nli_f_u/requirements.txt (line 1)) (2023.3)
[2023-09-20T08:09:00.323+0000] {process_utils.py:190} INFO - Requirement already satisfied: tzdata>=2022.1 in /home/***/.local/lib/python3.8/site-packages (from pandas->-r /tmp/venv3nli_f_u/requirements.txt (line 1)) (2023.3)
[2023-09-20T08:09:00.324+0000] {process_utils.py:190} INFO - Requirement already satisfied: numpy>=1.20.3 in /home/***/.local/lib/python3.8/site-packages (from pandas->-r /tmp/venv3nli_f_u/requirements.txt (line 1)) (1.24.4)
[2023-09-20T08:09:00.429+0000] {process_utils.py:190} INFO - Collecting contourpy>=1.0.1 (from matplotlib->-r /tmp/venv3nli_f_u/requirements.txt (line 2))
[2023-09-20T08:09:00.429+0000] {process_utils.py:190} INFO -   Obtaining dependency information for contourpy>=1.0.1 from https://files.pythonhosted.org/packages/9b/cb/08f884c4c2efd433a38876b1b8069bfecef3f2d21ff0ce635d455962f70f/contourpy-1.1.1-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata
[2023-09-20T08:09:00.430+0000] {process_utils.py:190} INFO -   Using cached contourpy-1.1.1-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (5.9 kB)
[2023-09-20T08:09:00.482+0000] {process_utils.py:190} INFO - Collecting cycler>=0.10 (from matplotlib->-r /tmp/venv3nli_f_u/requirements.txt (line 2))
[2023-09-20T08:09:00.483+0000] {process_utils.py:190} INFO -   Using cached cycler-0.11.0-py3-none-any.whl (6.4 kB)
[2023-09-20T08:09:00.583+0000] {process_utils.py:190} INFO - Collecting fonttools>=4.22.0 (from matplotlib->-r /tmp/venv3nli_f_u/requirements.txt (line 2))
[2023-09-20T08:09:00.583+0000] {process_utils.py:190} INFO -   Obtaining dependency information for fonttools>=4.22.0 from https://files.pythonhosted.org/packages/61/3f/68c99c2b5fb29f42ba22027d035ae08f0b2ea0178fbf00e7cef730b1a809/fonttools-4.42.1-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata
[2023-09-20T08:09:00.584+0000] {process_utils.py:190} INFO -   Using cached fonttools-4.42.1-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (150 kB)
[2023-09-20T08:09:00.670+0000] {process_utils.py:190} INFO - Collecting kiwisolver>=1.0.1 (from matplotlib->-r /tmp/venv3nli_f_u/requirements.txt (line 2))
[2023-09-20T08:09:00.671+0000] {process_utils.py:190} INFO -   Obtaining dependency information for kiwisolver>=1.0.1 from https://files.pythonhosted.org/packages/10/15/0c225864a00eb3dbd3d04b6465f331dc12a6e80673449166457b9566036d/kiwisolver-1.4.5-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata
[2023-09-20T08:09:00.671+0000] {process_utils.py:190} INFO -   Using cached kiwisolver-1.4.5-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (6.4 kB)
[2023-09-20T08:09:00.672+0000] {process_utils.py:190} INFO - Requirement already satisfied: packaging>=20.0 in /home/***/.local/lib/python3.8/site-packages (from matplotlib->-r /tmp/venv3nli_f_u/requirements.txt (line 2)) (23.1)
[2023-09-20T08:09:00.895+0000] {process_utils.py:190} INFO - Collecting pillow>=6.2.0 (from matplotlib->-r /tmp/venv3nli_f_u/requirements.txt (line 2))
[2023-09-20T08:09:00.896+0000] {process_utils.py:190} INFO -   Obtaining dependency information for pillow>=6.2.0 from https://files.pythonhosted.org/packages/2e/17/f19ad27effaf9b1ce6fe88d9f238d430293c486e33fdac71f037153cfd37/Pillow-10.0.1-cp38-cp38-manylinux_2_28_aarch64.whl.metadata
[2023-09-20T08:09:00.897+0000] {process_utils.py:190} INFO -   Using cached Pillow-10.0.1-cp38-cp38-manylinux_2_28_aarch64.whl.metadata (9.5 kB)
[2023-09-20T08:09:00.898+0000] {process_utils.py:190} INFO - Requirement already satisfied: pyparsing>=2.3.1 in /home/***/.local/lib/python3.8/site-packages (from matplotlib->-r /tmp/venv3nli_f_u/requirements.txt (line 2)) (3.1.1)
[2023-09-20T08:09:00.898+0000] {process_utils.py:190} INFO - Requirement already satisfied: importlib-resources>=3.2.0 in /home/***/.local/lib/python3.8/site-packages (from matplotlib->-r /tmp/venv3nli_f_u/requirements.txt (line 2)) (6.0.1)
[2023-09-20T08:09:00.930+0000] {process_utils.py:190} INFO - Requirement already satisfied: zipp>=3.1.0 in /home/***/.local/lib/python3.8/site-packages (from importlib-resources>=3.2.0->matplotlib->-r /tmp/venv3nli_f_u/requirements.txt (line 2)) (3.16.2)
[2023-09-20T08:09:00.941+0000] {process_utils.py:190} INFO - Requirement already satisfied: six>=1.5 in /home/***/.local/lib/python3.8/site-packages (from python-dateutil>=2.8.2->pandas->-r /tmp/venv3nli_f_u/requirements.txt (line 1)) (1.16.0)
[2023-09-20T08:09:01.016+0000] {process_utils.py:190} INFO - Using cached matplotlib-3.7.3-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (11.4 MB)
[2023-09-20T08:09:01.031+0000] {process_utils.py:190} INFO - Using cached contourpy-1.1.1-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (285 kB)
[2023-09-20T08:09:01.040+0000] {process_utils.py:190} INFO - Using cached fonttools-4.42.1-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (4.5 MB)
[2023-09-20T08:09:01.047+0000] {process_utils.py:190} INFO - Using cached kiwisolver-1.4.5-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (1.4 MB)
[2023-09-20T08:09:01.051+0000] {process_utils.py:190} INFO - Using cached Pillow-10.0.1-cp38-cp38-manylinux_2_28_aarch64.whl (3.5 MB)
[2023-09-20T08:09:03.191+0000] {process_utils.py:190} INFO - Installing collected packages: pillow, kiwisolver, fonttools, cycler, contourpy, matplotlib
[2023-09-20T08:09:04.632+0000] {process_utils.py:190} INFO - Successfully installed contourpy-1.1.1 cycler-0.11.0 fonttools-4.42.1 kiwisolver-1.4.5 matplotlib-3.7.3 pillow-10.0.1
[2023-09-20T08:09:04.939+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venv3nli_f_u/bin/python /tmp/venv3nli_f_u/script.py /tmp/venv3nli_f_u/script.in /tmp/venv3nli_f_u/script.out /tmp/venv3nli_f_u/string_args.txt /tmp/venv3nli_f_u/termination.log
[2023-09-20T08:09:04.946+0000] {process_utils.py:186} INFO - Output:
[2023-09-20T08:09:06.311+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-09-20T08:09:06.320+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=create_chart, execution_date=20230920T080855, start_date=20230920T080858, end_date=20230920T080906
[2023-09-20T08:09:06.334+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-20T08:09:06.344+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
